_translations/bg.yml

```diff
@@ -563,7 +563,7 @@ bg:
     donation: "Направете дарение"
     donationtxt: "Най-лесният начин да помогнете е като  <a href=\"https://bitcoinfoundation.org/about/donate/\">дарите</a> малко биткойни на Фондация Биткойн. Също така може директно да финансирате проекти, свързани с Биткойн, които смятате, че ще са от полза в бъдещето."
     nonprofit: "Неправителствени организации"
-    nonprofittxt: "Bitcoin фондация< /a> и много други организации с нестопанска цел< /a> са посветени на защитата и насърчаването Bitcoin. Можете да се помогне на тези групи от присъединяване към тях и участие в проектите им, дискусии и Събития."
+    nonprofittxt: "Bitcoin фондация и много други организации с нестопанска цел са посветени на защитата и насърчаването Bitcoin. Можете да се помогне на тези групи от присъединяване към тях и участие в проектите им, дискусии и Събития."
```

Comment:
@harding This one appears to have two links missing (see the corresponding English text).